.circleci/config.yml

```diff
@@ -65,6 +65,8 @@ commands:
       - run:
           name: embuilder build ALL
           command: |
+            # TODO(sbc): These currently seems to lock up windows.
```

Comment:
would be good to at least check this on linux, as it is a painful problem (i did a long regression now through a range where this failed, and had to constantly apply and tweak a patch)